### PR TITLE
Use target="_blank" instead of class="_blank"

### PR DESCRIPTION
**Pull request by PululuK**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop 
| Description?  | Currently both `class="_blank"` and `target="_blank"` are proper. This is because https://github.com/PrestaShop/PrestaShop/blob/develop/js/tools.js#L761. But today `target` attribute is compatible with most browsers  target_black_compatiility See :  https://developer.mozilla.org/fr/docs/Web/HTML/Element/a#Compatibilit%C3%A9_des_navigateurs and https://www.w3schools.com/tags/tag_a.asp
| Type?         |  improvement 
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | N/A.
| How to test?  | N/A ??.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/19710)

### CODE REVIEW

**Comment by Progi1984** [comment]
@PrestaShop/prestashop-qa Will this PR break your automated tests ?

**Review by Progi1984 — DISMISSED** [review]

**Comment by boubkerbribri** [comment]
> @PrestaShop/prestashop-qa Will this PR break your automated tests ?

This will not break our tests, thank you.

**Comment by Progi1984** [comment]
Thanks @boubkerbribri 

@PrestaShop/prestashop-qa Do you need test for that ?

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Backup/Blocks/backup_info.html.twig [hunk 1]
```diff
@@ -27,7 +27,7 @@
   <div class="alert alert-info" role="alert">
     {% if isHostMode %}
       <h4>{{ 'How to restore a database backup'|trans({}, 'Admin.Advparameters.Notification') }}</h4>
-      <p>{{ 'If you need to restore a database backup, we invite you to subscribe to a [1][2]technical support plan[/2][/1].'|trans({'[1]': '<strong>',  '[/1]': '</strong>', '[2]': '<a class="_blank" href="https://addons.prestashop.com/support/16298-support-essentiel-plan.html">', '[/2]': '</a>'}, 'Admin.Advparameters.Notification')|raw }}</p>
+      <p>{{ 'If you need to restore a database backup, we invite you to subscribe to a [1][2]technical support plan[/2][/1].'|trans({'[1]': '<strong>',  '[/1]': '</strong>', '[2]': '<a target="_blank" href="https://addons.prestashop.com/support/16298-support-essentiel-plan.html">', '[/2]': '</a>'}, 'Admin.Advparameters.Notification')|raw }}</p>
       <p>{{ 'Our team will take care of restoring your database safely.'|trans({}, 'Admin.Advparameters.Notification') }}</p>
       <br>
       <p>{{ 'Why can\'t I restore it by myself?'|trans({}, 'Admin.Advparameters.Notification') }}</p>
```
Comment (Junebyun) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Backup/Blocks/backup_info.html.twig:33]:
```suggestion
      <p>{{ 'Why can't I restore it by myself?'|trans({}, 'Admin.Advparameters.Notification') }}</p>
```

Comment (PululuK) [reply src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Backup/Blocks/backup_info.html.twig:33]: Hi @Junebyun  😅

- It has nothing to do with my PR
- It is an escape sequences, it should not be removed http://www.hackingwithphp.com/2/6/2/escape-sequences

Thank you anyway

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
For security reasons, you need to add `rel="noopener noreferrer nofollow"` when using the `targe="_blank"`, Could you please add this?

**Comment by PululuK** [comment]
> For security reasons, you need to add `rel="noopener noreferrer nofollow"` when using the `targe="_blank"`, Could you please add this?

I don't think it's mandatory in the admin pages ... 🤔
https://developer.mozilla.org/fr/docs/Web/HTML/Element/a#S%C3%A9curit%C3%A9_et_vie_priv%C3%A9e and https://developer.mozilla.org/fr/docs/Web/Security/Referer_header:_privacy_and_security_concerns

**Review by LouiseBonnard — DISMISSED** [review]

**Comment by PierreRambaud** [comment]
> > For security reasons, you need to add `rel="noopener noreferrer nofollow"` when using the `targe="_blank"`, Could you please add this?
> 
> I don't think it's mandatory in the admin pages ... 
> https://developer.mozilla.org/fr/docs/Web/HTML/Element/a#S%C3%A9curit%C3%A9_et_vie_priv%C3%A9e

The aim here is to have a single way to use `target="_blank"` and help all reviewers to don't forget this one in case of pointing to an external resource.
More, the `getAdminLink` function can be modified by a module and target an external resource.

File: themes/classic/templates/_partials/footer.tpl [hunk 2]
```diff
@@ -45,7 +45,7 @@
       <div class="col-md-12">
         <p class="text-sm-center">
           {block name='copyright_link'}
-            <a class="_blank" href="https://www.prestashop.com" target="_blank" rel="nofollow">
+            <a href="https://www.prestashop.com" target="_blank" rel="nofollow">
```
Comment (Progi1984) [inline themes/classic/templates/_partials/footer.tpl]:
```suggestion
            <a href="https://www.prestashop.com" target="_blank" rel="noopener noreferrer nofollow">
```

**Review by Progi1984 — COMMENTED** [review]
A last change before my approve

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by Progi1984** [comment]
@PrestaShop/qa-functional Need QA for that ?

**Review by Junebyun — APPROVED** [review]

**Comment by SimonGrn** [comment]
> @PrestaShop/qa-functional Need QA for that ?

I don't see any meaningful changes to the code, but we'll do an exploratory pass since it's kinda complicated to find all the links @PululuK changed.

**Comment by SimonGrn** [comment]
> @PrestaShop/qa-functional Need QA for that ?

I don't see any "dangerous" changes to the code, but we'll do an exploratory pass since it's kinda complicated to find all the links @PululuK changed.

**Review by NeOMakinG — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by SD1982** [comment]
LGTM Thanks @PululuK !

**Comment by NeOMakinG** [comment]
@PululuK Thanks!

**Pull request merged**